packages/website/src/docs/textarea.mdx

```diff
@@ -1,23 +1,42 @@
 ---
 title: Textarea
 component: Textarea
+description: Useful for allowing users to enter, update and select text.
 ---
 
-```js live=true
+```js live
 <Textarea placeholder="Leave your comment here" />
 ```
 
-```js live=true
-<Textarea size={1} placeholder="Leave your comment here" />
+### size
+
+<Prop isOptional default="0">
+  0 | 1
+</Prop>
+
+There are 2 size options to choose from.
```

Comment:
2 sizes to choose from

Reply:
It says "2 size options". It's consistent with some of the `Button` docs. But could re-write all of them. 

Do you think removing `options` is better?

Reply:
Yeah I feel it's not needed, I commented on all of them. It's like for variants you don't say: "there are 3 variant options"